### PR TITLE
Revised SDL-270 - Add means to launch with optimum appHMIType -

### PR DESCRIPTION
**Pull request by Kazuki-Sugimoto**
Taken over from his predecessor, Shohei-Kawano.
Added use case according to [comments](https://github.com/smartdevicelink/sdl_evolution/issues/892#issuecomment-567089987).

### CODE REVIEW

**Comment by Kazuki-Sugimoto** [comment]
@theresalech -san
Hi, Could you review this proposal?

**Comment by Shohei-Kawano** [comment]
@theresalech -san
I know this PR. I have a lot of my work and I have him help.

@Kazuki-Sugimoto -san
Could you add my name as a co-authoring?

File: proposals/0270-Add-means-to-launch-with-optimum-appHMIType.md [hunk 1]
```diff
@@ -99,7 +99,18 @@ Add `launchAppHMIType` to the RPC below.
...
+
+For example, a navigation app with music playback function:
+
+1. When touch the app icon from the connectivity function screen to launch it (specify Navigation in launchAppHMIType), the navigation screen using video projection mode is displayed.
+
+2. When touch the app icon from the audio function screen to launch it (specify Media in launchAppHMIType), the music playback screen using the media template is displayed.
+
+3. When touch the app icon from the connectivity function screen to launch it (specify Navigation in launchAppHMIType) while the music playback screen used the media template is displayed on the app, the navigation screen using video projection mode is displayed while playing music.
+
+4. When touch the app icon from the audio function screen to launch it (specify Media in launchAppHMIType) during navigation by the app, the music playback screen using the media template is displayed during navigation.
```
[the remark this answers is not in the document]
Comment (Kazuki-Sugimoto) [reply proposals/0270-Add-means-to-launch-with-optimum-appHMIType.md]: Does it make sense to use "the user" only this suggestion?

**Comment by Kazuki-Sugimoto** [comment]
@theresalech -san
Thank you for the review.
I replied your suggestion, please check.

File: proposals/0270-Add-means-to-launch-with-optimum-appHMIType.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (theresalech) [reply proposals/0270-Add-means-to-launch-with-optimum-appHMIType.md]: This can be "When a user touches..." as well, thanks!

**Comment by theresalech** [comment]
@Kazuki-Sugimoto I have replied to your question.  Please review and address that feedback, as well as the suggestions to add Kawano-san as a co-author and to include your findings from investigating the alternative options suggested by the Project Maintainer, either in the "Alternatives Considered" section, or as a new solution.  Please see [this comment](https://github.com/smartdevicelink/sdl_evolution/issues/892#issuecomment-567089987).  Thanks!

**Comment by Kazuki-Sugimoto** [comment]
@theresalech -san
All of your suggestions have been addressed.

**Comment by E-SAITO-TMC** [comment]
This PR is toyota's request.
We hope to include in SDL Core 6.1.

**Comment by joeygrover** [comment]
Hi @E-SAITO-TMC,
The 6.1 Core release has already been planned and voted on by the SDLC Steering Committee. Since this proposal had not been reviewed or accepted, it was not possible to be included during the planning of that release. Once this proposal is reviewed and if accepted, it can be brought in as a priority for the upcoming Core release (6.2) in October. Thank you for your understanding and we can have further discussions in the next Steering Committee meeting if needed!

**Pull request merged**